examples/demo-rollup/localsetup.md

```diff
@@ -0,0 +1,47 @@
+## Setting up SDK to run locally
+
+* Install docker https://www.docker.com
+* switch to the `demo-rollup` directory
+* Start the celestia services locally
+```
+make clean
+make start
+```
+* The above command should also configure your local setup so you should see some changes stashed
+```
+$ git status
+..
+..
+	modified:   ../const-rollup-config/src/lib.rs
+	modified:   rollup_config.toml
+```
+* Start the demo-rollup in a different tab
+```
+$ cargo +nightly run
+```
+* You should see the demo-rollup app consuming blocks from the docker container's celestia node
+```
+2023-06-07T10:03:25.473920Z  INFO jupiter::da_service: Fetching header at height=1...
+2023-06-07T10:03:25.496853Z  INFO sov_demo_rollup: Received 0 blobs
+2023-06-07T10:03:25.497700Z  INFO sov_demo_rollup: Requesting data for height 2 and prev_state_root 0xa96745d3184e54d098982daf44923d84c358800bd22c1864734ccb978027a670
+2023-06-07T10:03:25.497719Z  INFO jupiter::da_service: Fetching header at height=2...
+2023-06-07T10:03:25.505412Z  INFO sov_demo_rollup: Received 0 blobs
+2023-06-07T10:03:25.505992Z  INFO sov_demo_rollup: Requesting data for height 3 and prev_state_root 0xa96745d3184e54d098982daf44923d84c358800bd22c1864734ccb978027a670
+2023-06-07T10:03:25.506003Z  INFO jupiter::da_service: Fetching header at height=3...
+2023-06-07T10:03:25.511237Z  INFO sov_demo_rollup: Received 0 blobs
+2023-06-07T10:03:25.511815Z  INFO sov_demo_rollup: Requesting data for height 4 and prev_state_root 0xa96745d3184e54d098982daf44923d84c358800bd22c1864734ccb978027a670
+```
+* Run the test transaction command, which creates a token
+```
+make test-create-token 
```

Comment:
Same thing here. Eventually (maybe after `clap` support is added) we should try to explain *what* to do, not just how to use the `Makefile` to do it.